test-project/Assets/EditmodeTests/GameObjectRepresentation/MonoBehaviourActivationManager/ActivationManagerTestBase.cs

```diff
@@ -0,0 +1,48 @@
+using Improbable.Gdk.Core;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.GameObjectRepresentation.EditModeTests.MonoBehaviourActivationManagerTests
+{
+    public abstract class ActivationManagerTestBase<TBehaviour> where TBehaviour : MonoBehaviour
```

Comment:
Given that this is used elsewhere with a more than one behaviour (hence why `PopulateBehaviours` is virtual) - why not remove the generic argument and make `PopulateBehaviours` fully abstract?

Reply:
was unsure how "DRY" it should get, will have another think on it